igor-web/igor-web.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
 
     compile 'org.yaml:snakeyaml:1.15'
     compile 'com.squareup.retrofit:converter-simplexml:1.9.0'
```

Comment:
I would love to drop that dependency, as it is no longer used in the Jenkins model classes. However annotations from simplexml are still used in the Travis model classes, which I find a bit surprising since the Travis API seems JSON based and already using Jackson. @gardleopard do you have more context on this? I suspect these annotations are actually unused but I don't have a good way to test that theory